### DISCUSSION TITLE
同样的配置电脑端可以用，安卓版NG不能用，

### DISCUSSION THREAD

**Discussion by madesun** [2dust/v2rayNG#1370] [Q&A]
同样的配置电脑端可以用，安卓版NG不能用，

**Comment by Johnny-dot** [comment]
我也是，暂时还是只能降版本用
https://www.apkmirror.com/apk/captainiron/v2rayng/v2rayng-1-4-13-release/v2rayng-1-4-13-2-android-apk-download/download/

**Reply by RealJeremy24** [reply]
谢谢兄弟，不过最新版不行是什么

**Comment by robin12jbj** [comment]
设置分应用代理或者全局代理了吗？
